### PR TITLE
more consistency for master

### PR DESCRIPTION
**Pull request by OskarStark**
Its much easier to review commit by commit

Todo before merging this PR:
- [x] Merge https://github.com/symfony/symfony-docs/pull/11243
- [x] Upmerge to `master`
- [x] Rebase this PR

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
Nice!

**Comment by OskarStark** [comment]
> Nice!

Spotted by "the tool" 😃

**Comment by OskarStark** [comment]
Please wait while merging this! I need to fix sth. else

**Comment by javiereguiluz** [comment]
Don't worry, I'm not going to merge this. I'm going to leave it to you, so you can handle it as soon as you are granted the merge permission 🎉

**Comment by OskarStark** [comment]
Todo list complete, please review again, thanks!

File: reference/forms/types/options/attr_translation_parameters.rst.inc [hunk 1]
```diff
@@ -16,23 +16,21 @@ Given this translation message:
...
+    $builder->add('id', null, [
+        'attr' => [
             'placeholder' => 'form.order.id.placeholder',
             'title' => 'form.order.id.title',
-        ),
+        ],
         'attr_translation_parameters' => [
-            '%company%' => 'ACME Inc.'
+            '%company%' => 'ACME Inc.',
         ]
```
Comment (xabbuh) [inline reference/forms/types/options/attr_translation_parameters.rst.inc]: Add a traling comma here too?

File: reference/forms/types/options/help_translation_parameters.rst.inc [hunk 2]
```diff
@@ -15,19 +15,17 @@ Given this translation message:
...
 
-.. code-block:: php
-
-    $builder->add('id', null, array(
+    $builder->add('id', null, [
         'help' => 'form.order.id.help',
         'help_translation_parameters' => [
-            '%company%' => 'ACME Inc.'
+            '%company%' => 'ACME Inc.',
         ]
```
Comment (xabbuh) [inline reference/forms/types/options/help_translation_parameters.rst.inc]: same here

File: reference/forms/types/options/label_translation_parameters.rst.inc [hunk 3]
```diff
@@ -15,19 +15,17 @@ Given this translation message:
...
 
-.. code-block:: php
-
-    $builder->add('id', null, array(
+    $builder->add('id', null, [
         'label' => 'form.order.id',
         'label_translation_parameters' => [
-            '%company%' => 'ACME Inc.'
+            '%company%' => 'ACME Inc.',
         ]
```
Comment (xabbuh) [inline reference/forms/types/options/label_translation_parameters.rst.inc]: and here

**Review by xabbuh — APPROVED** [review]

**Pull request merged**